MapEditorReborn/Commands/ToolgunCommands/CreateObject.cs

```diff
@@ -145,10 +150,27 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
             Vector3 pos = ev.Position;
             parsedEnum = ev.ObjectType;
 
-            ToolGunHandler.SpawnObject(pos, parsedEnum);
+            GameObject gameObject = ToolGunHandler.SpawnObject(pos, parsedEnum);
             response = $"{parsedEnum} has been successfully spawned!";
 
+            if (MapEditorReborn.Singleton.Config.AutoSelect)
+                TrySelectObject(player, SpawnedObjects.FirstOrDefault(o => o.gameObject == gameObject));
```

Comment:
Why can't you directly pass the gameObject?

Reply:
Because i need MapEditorObject to call this method that is why i find it in SpawnedObjects using gameObject